services/ma-myocyteele/docker/ubuntu/Dockerfile

```diff
@@ -0,0 +1,106 @@
+FROM ${DOCKER_REGISTRY:-itisfoundation}/mat2019b_ubu1804:0.1.0 as base
```

Comment:
this is the matlab image right?
to follow the conventions since it is not anymore the ubuntu image, please move everything located in /ubuntu to /custom folder so it is consistent. you will need to adapt the Makefile, compose or whatever is ling to /ubuntu.

Reply:
but here the base image is PUBLIC, i.e. can only have itisfoundation as namespace

Reply:
@sanderegg actually I noticed that other services have base images from the **internal** repository which IMO should not happen. because
a) the CI needs credentials to access private repo
b) a PUBLIC image gives access to a PRIVATE base

Reply:
@sanderegg changed the folder naming - let me know if I did it wrong

Reply:
> @sanderegg but here the base image is PUBLIC, i.e. can only have itisfoundation as namespace

@pcrespov this has nothing to do with public/private.. this has to do with the fact that the cookiecutter generates a cookie for some pre-defined images (e.g. python, ubuntu, alpine) and they go in their respective sub-folder. When deviating from this it should go in ```custom``` subfolder. That is all.

Reply:
> @sanderegg actually I noticed that other services have base images from the **internal** registry (i.e. masu) which IMO should **not** happen. because
> a) the CI needs credentials to access private repo
> b) a PUBLIC image gives access to a PRIVATE base
@pcrespov I agree that is bad. since masu is not accessible. anything private should not be in this repo.